### ISSUE TITLE
Variable BookmarksFile being overwritten causing error?

### ISSUE THREAD

**Issue by teunissenstefan** [BrodieRobertson/bookmenu#2]
When I run the script without changes and try to open a bookmark I get the following output:
```
./bookmenu: line 146: [: too many arguments
sort: cannot read: 'duckduckgo:duckduckgo.com': No such file or directory
grep: duckduckgo:duckduckgo.com: No such file or directory
grep: ebay:www.ebay.com: No such file or directory
grep: paypal:www.paypal.com: No such file or directory
grep: amazon:www.amazon.com: No such file or directory
grep: youtube:www.youtube.com: No such file or directory
grep: wikipedia:www.wikipedia.com: No such file or directory
grep: github:www.github.com: No such file or directory
grep: reddit:www.reddit.com: No such file or directory
```
So I took a look in the script and saw that you are overwriting the **BookmarksFile** variable which (I think) should point to `$HOME/.config/bookmenu/bookmarks` on line 16.
It is being overwritten on line 21.

After commenting out or removing lines 21-28, The script works without any problems.
```
 21 #BookmarksFile="duckduckgo:duckduckgo.com
 22 #ebay:www.ebay.com
 23 #paypal:www.paypal.com
 24 #amazon:www.amazon.com
 25 #youtube:www.youtube.com
 26 #wikipedia:www.wikipedia.com
 27 #github:www.github.com
 28 #reddit:www.reddit.com"
```

Could've submitted a pull request but I'm not sure if this was done intentional and just isn't working on my system, or if it is indeed an error.

**Comment by BrodieRobertson** [comment]
I know what I did, I was changing around some variable names and that was originally a string for default bookmarks. I'll fix that

**Comment by BrodieRobertson** [comment]
This should now be fixed in commit 3fd3895
